src/steps/os/linux.rs

```diff
@@ -52,6 +53,7 @@ impl Distribution {
 
         Ok(match id {
             Some("alpine") => Distribution::Alpine,
+            Some("wolfi") => Distribution::Wolfi,
```

Comment:
Thanks for the suggestion, removed as it is unique

Reply:
Seems that we are in the reverse direction, we should remove the check on the `ID_LIKE` field rather than the one on `ID`, and the current implementation won't work:

```
Location:
    /rustc/82e1608dfa6e0b5569232559e3d385fea5a93112/library/core/src/convert/mod.rs:757:9
thread 'steps::os::linux::tests::test_wolfi' panicked at src/steps/os/linux.rs:1031:57:
called `Result::unwrap()` on an `Err` value: Unknown Linux Distribution
```

Reply:
Accidentally purged the wrong one. This is resolved now. Thank you for checking :)